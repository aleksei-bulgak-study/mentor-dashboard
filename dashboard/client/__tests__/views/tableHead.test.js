import React from 'react';
import { shallow } from 'enzyme';
import TableHead from '../../src/views/tableHead';

describe('TableHead', () => {
  test('should match snapshot', () => {
    const students = [
      'student', 'student2', 'student3',
    ];
    const wrapper = shallow(<TableHead students={students} />);
    expect(wrapper).toMatchSnapshot();
  });


  test('should match snapshot with empty list of students', () => {
    const students = [];
    const wrapper = shallow(<TableHead students={students} />);
    expect(wrapper).toMatchSnapshot();
  });
});
