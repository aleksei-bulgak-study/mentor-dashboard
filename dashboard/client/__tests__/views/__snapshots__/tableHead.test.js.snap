// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHead should match snapshot 1`] = `
<thead>
  <tr>
    <th />
    <Student
      key="student"
      name="student"
    />
    <Student
      key="student2"
      name="student2"
    />
    <Student
      key="student3"
      name="student3"
    />
  </tr>
</thead>
`;

exports[`TableHead should match snapshot with empty list of students 1`] = `
<thead>
  <tr>
    <th />
  </tr>
</thead>
`;
